### PR TITLE
First iteration of Dart math-methods entry

### PR DESCRIPTION
**Pull request by dalling123**
### Description

- Added entry for Dart math methods.

### Issue Solved

- Closes Issue #4243 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/dart/concepts/math-methods/math-methods.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Perform trigonometric calculations and return the larger or smaller of two numbers using `max()` and `min()` methods.  
+
+```dart
+var result = cos(pi); // Calculates cosine of pi radians
+print(result); // Prints -1
+
+var highestNumber = max(30, 45);
+print(highestNumber); // Prints 45
+```
```
Comment (avdhoottt) [inline content/dart/concepts/math-methods/math-methods.md:56]: Remove this section as there will be separate term entries for these methods.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
An overall comment for you. Try to add more content to it, for example, you can add a table that includes the methods and their short description (Don't go too much in deep as I said there will be separate entries)

**Comment by avdhoottt** [comment]
@dalling123 Are you still working on this PR?

**Comment by dalling123** [comment]
Hi Avdhoot,

Yes I am! Just on holiday this week - I can complete next week if thats
okay! Best,

Dylan

**Comment by avdhoottt** [comment]
@dalling123 are you available to make the changes now?

**Review by avdhoottt — APPROVED** [review]
Looks good for the second review.

**Comment by dalling123** [comment]
Hello @avdhoottt and @ishg-153,
What next steps are needed here?

**Review by ishg-153 — COMMENTED** [review]
@dalling123 Great Work! Ready to merge!

**Review by ishg-153 — APPROVED** [review]

**Pull request merged**